.github/workflows/build.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
           check-latest: true
 
       # will use the latest release available for ko
-      - uses: imjasonh/setup-ko@ace48d793556083a76f1e3e6068850c1f4a369aa # v0.6
+      - uses: ko-build/setup-ko@ace48d793556083a76f1e3e6068850c1f4a369aa # v0.6
```

Comment:
I'd suggest to make these changes as part of a different PR

Reply:
Oddly, this failed to build in our private fork as the setup-ko action has moved -- not sure why it works here! I'll move them to a different PR.

Reply:
https://github.com/sigstore/policy-controller/pull/1328